src/documentation/tfconnect/tfconnect_installation.md

```diff
@@ -0,0 +1,108 @@
+# ThreeFold Connect Installation
+
+## Introduction
+
+We show how to install the ThreeFold Connect app.
+
+![](./img/tfconnect_8.png)
+
+## Links
+
+The ThreeFold Connect app is available for both iOS and Android devices.
+
+- [Google Store Link](https://play.google.com/store/apps/details?id=org.jimber.threebotlogin)
+- [Apple Store](https://itunes.apple.com/be/app/3bot-login/id1459845885?l=nl&mt=8)
+
+## Installation
+
+Either use the links above, or search for the ThreeFold Connect app on the Apple Store or the Google Play store. If you want to leave a 5 star review of the app, no one here will stop you!
+
+Click on `Install`. 
+
+![](./img/tfconnect_9.png)
+
+Once the app is installed, click `Open`.
+
+![](./img/tfconnect_10.png)
+
+Once you are in the application, you will see some introduction pages to help you familiarize yourself with the TF Connect app.
+
+![](./img/tfconnect_11.png)
+
+You will then be asked to read and accept ThreeFold's Terms and Conditions.
+
+![](./img/tfconnect_12.png)
+
+## Create Account
+
+Once you've accepted the Terms and Conditions, you can either *Create Account* or *Log In*. For now, we will show how to creae a new account. Later in the guide, we will show you how to log in, which can be used to recover your account.
+
+![](./img/tfconnect_13.png)
+
+To create a new account, you will be asked to choose a `Threefold Connect Id`. This ID will be used, as well as the seed phrase, when you want to log in after having logged out of your account.
+
+You will also need to add a valid email address. You will need to access your email and confirm the validation email from ThreeFold to use properly the TF Connect app wallet.
+
+You will be shown your seed phrase. Keep this in a secure and offline place. You will need the 3bot ID and the seed phrase to recover your account. This seed phrase is of utmost importance. Do not lose it nor give it to anyone.
+
+Once you've hit Next, you will be asked to write down 3 random words of your seed phrase. This is a necessary step to ensure you have taken the time to write down your seed phrase.
+
+Then, you'll be asked to confirm your TF 3bot name and the associated email.
+
+![](./img/tfconnect_14.png)
+
+Finally, you will be asked to choose a 4-digit PIN. This will be needed to use the ThreeFold Connect app. If you ever forget this 4-digit PIN, you will need to recover your account from your 3bot name and your seed phrase. You will need to confirm the new pin in the next step.
+
+That's it! You've created your ThreeFold Connect account.
+
+## Email Verification
```

Comment:
There is also KYC on the app itself too; not only dashboard now, which should be covered in the app manual as well.

we now have:
- Email verification
- Mobile number verification
- KYC verification


all should be covered with screen shots too

Reply:
Done! LMK if anything else needs to be done. Thanks.